setup.py

```diff
@@ -91,7 +74,7 @@
             'sphinx-rtd-theme',
         ]
     },
-    'packages': [name],  # Manually specify here, update after autogen
+    'packages': setuptools.find_packages(),
```

Comment:
this is always a bit fraught... it would be good to check exactly what goes in, and potentially add some `exclude`. Moving to a `src/` layout helps this _a lot_, but is probably out of scope for this.

Reply:
Yeah, let's push any major layout changes that a different PR :)

Reply:
Was this motivated by something? I think the comment is still important, as *the package list made here won't include the autogenerated python subpackages* no matter which method we choose. I see the code of the pythreejs simply removed this important step: https://github.com/jupyter-widgets/pythreejs/pull/342#discussion_r575259828

Reply:
I seem to have dropped the ball on this; I tested things locally but something must have been cached. I'll revert the change.